src/components/Select/Select.js

```diff
@@ -23,37 +24,41 @@ export const Select = ({
   formTouched,
   labelCopy,
   name,
+  initialValue,
```

Comment:
I'm not seeing `initialValue` take affect:

![image](https://user-images.githubusercontent.com/142403/71848541-931a1d80-3084-11ea-8e49-647c05da8894.png)